extras/indexing/src/main/java/org/apache/rya/api/client/accumulo/AccumuloCreatePCJ.java

```diff
@@ -139,16 +138,15 @@ private void updateFluoApp(final String ryaInstance, final String fluoAppName, f
 
         // Connect to the Fluo application that is updating this instance's PCJs.
         final AccumuloConnectionDetails cd = super.getAccumuloConnectionDetails();
-        final FluoClient fluoClient = new FluoClientFactory().connect(
+        try(final FluoClient fluoClient = new FluoClientFactory().connect(
                 cd.getUsername(),
                 new String(cd.getPassword()),
                 cd.getInstanceName(),
                 cd.getZookeepers(),
-                fluoAppName);
-
-        // Initialize the PCJ within the Fluo application.
-        final org.apache.rya.indexing.pcj.fluo.api.CreatePcj fluoCreatePcj = new org.apache.rya.indexing.pcj.fluo.api.CreatePcj();
-        fluoCreatePcj.withRyaIntegration(pcjId, pcjStorage, fluoClient, getConnector(), ryaInstance);
+                fluoAppName);) {
+            // Initialize the PCJ within the Fluo application.
+            final org.apache.rya.indexing.pcj.fluo.api.CreatePcj fluoCreatePcj = new org.apache.rya.indexing.pcj.fluo.api.CreatePcj();
```

Comment:
No. The class name CreatePcj is used twice in the project. This class implements one, which it imports, so this one can't be.